src/server/game/Warden/WardenWin.cpp

```diff
@@ -336,12 +448,14 @@ void WardenWin::RequestData()
 
     _dataSent = true;
 
+#if defined(ENABLE_EXTRAS) && defined(ENABLE_EXTRA_LOGS)
     std::stringstream stream;
     stream << "Sent check id's: ";
-    for (std::list<uint16>::iterator itr = _currentChecks.begin(); itr != _currentChecks.end(); ++itr)
-        stream << *itr << " ";
+    for (uint16 checkId : _currentChecks)
```

Comment:
Upper case on _CurrentChecks

Reply:
... this is funny... making improved but being tossed to do whole package... grr idiots...